dashboard/test/models/script_test.rb

```diff
@@ -70,71 +70,6 @@ def populate_cache_and_disconnect_db
     ActiveRecord::Base.connection.stubs(:execute).raises 'Database disconnected'
   end
 
-  test 'can create course version when seeding a unit' do
-    unit = Script.add_unit(
-      {name: 'unit-name', family_name: 'family', version_year: 'unversioned', published_state: SharedCourseConstants::PUBLISHED_STATE.preview, is_course: true},
-      {}
-    )
-    course_version = unit.course_version
-    assert_not_nil course_version
-    assert_equal 'unversioned', course_version.key
-    assert_equal 'family', course_version.course_offering&.key
-    assert_equal SharedCourseConstants::PUBLISHED_STATE.preview, course_version.published_state
-  end
-
-  test 'cannot rename a unit without a new_name' do
-    l = create :level
-    dsl = <<-UNIT
-      lesson 'Lesson1', display_name: 'Lesson1'
-      level '#{l.name}'
-    UNIT
-    old_unit = Script.add_unit(
-      {name: 'old unit name'},
-      ScriptDSL.parse(dsl, 'a filename')[0][:lesson_groups]
-    )
-    assert_equal 'old unit name', old_unit.name
-
-    new_unit = Script.add_unit(
-      {name: 'new unit name'},
-      ScriptDSL.parse(dsl, 'a filename')[0][:lesson_groups]
-    )
-    assert_equal 'new unit name', new_unit.name
-
-    # a new unit was created
-    refute_equal old_unit.id, new_unit.id
-  end
-
-  test 'can rename a unit between original name and new_name' do
-    l = create :level
-    dsl = <<-UNIT
-      lesson 'Lesson1', display_name: 'Lesson1'
-      level '#{l.name}'
-    UNIT
-    old_unit = Script.add_unit(
-      {name: 'old unit name', new_name: 'new unit name'},
-      ScriptDSL.parse(dsl, 'a filename')[0][:lesson_groups]
-    )
-    assert_equal 'old unit name', old_unit.name
-
-    new_unit = Script.add_unit(
-      {name: 'new unit name', new_name: 'new unit name'},
-      ScriptDSL.parse(dsl, 'a filename')[0][:lesson_groups]
-    )
-    assert_equal 'new unit name', new_unit.name
-
-    # the old unit was renamed
-    assert_equal old_unit.id, new_unit.id
-
-    old_unit = Script.add_unit(
-      {name: 'old unit name', new_name: 'new unit name'},
-      ScriptDSL.parse(dsl, 'a filename')[0][:lesson_groups]
-    )
-    assert_equal 'old unit name', old_unit.name
-
-    # the unit was renamed back to the old name
-    assert_equal old_unit.id, new_unit.id
-  end
```

Comment:
the ability to rename units (which we used in 2017 to rename e.g. coursea to coursea-2017) is not something we support for migrated units, so no equivalent test coverage is needed for migrated units.